controllers/ReservationSystem.php

```diff
@@ -336,6 +336,11 @@ public function cancelBooking($channelId, $bookingId)
         return $result->booking;
     }
 
+    public function deleteBooking($bookingId, $channelId)
+    {
+        $result = $this->TourCMSAgent->delete_booking($bookingId, $channelId);
+        return $result->booking;
```

Comment:
What are you handling here?
All API request should be checked against error field